src/ws2st/meta/io.cpp

```diff
@@ -0,0 +1,232 @@
+#include "meta.hpp"
+
+#include <utility>
+
+namespace wash {
+
+namespace refactor {
+
+namespace meta {
+
+    DeclarationMatcher DefineForceAccessFnMatcher = traverse(TK_IgnoreUnlessSpelledInSource, 
+        cxxRecordDecl(hasName("_define_force_access_fn")).bind("decl")
+    );
+
+    void DefineForceAccessFns(const MatchFinder::MatchResult& Result, Replacements& Replace) {
+        const auto *decl = Result.Nodes.getNodeAs<CXXRecordDecl>("decl");
+        if (!decl) {
+            throw std::runtime_error("Missing decl");
+        }
+        std::string output_str = "";
+        for (auto type :  std::vector<std::string> {"scalar", "vector"}) {
```

Comment:
Minor: instead of iterating over a vector of both string here, would prefer the loop body to be taken out into a new function. Then can just call this new function once for `scalar` and once for `vector`

Reply:
Yeah, implemented this